util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialConfiguration.java

```diff
@@ -60,11 +60,15 @@
       new HadoopConfigurationProperty<>(
           ".auth.service.account.email", /* defaultValue= */ null, ".service.account.auth.email");
 
-  /** Key suffix used to specify private key id for the service account. */
+  /**
```

Comment:
Use Google Java Format to format changed lines in this PR: https://github.com/google/google-java-format/blob/master/scripts/google-java-format-diff.py